go.mod

```diff
@@ -9,4 +9,4 @@ require (
 	google.golang.org/grpc v1.47.0
 )
 
-go 1.14
+go 1.16
```

Comment:
Can we also update the [Dockerfile](https://github.com/ExpediaGroup/mittens/blob/cc68e6981437bd59bad419d3b4eaa11137bb045f/Dockerfile#L1) and the reference in the [readme](https://github.com/ExpediaGroup/mittens/blob/cc68e6981437bd59bad419d3b4eaa11137bb045f/docs/installation/building.md#binary)? :)